compiler/modules.nim

```diff
@@ -133,11 +133,12 @@ proc newModule(graph: ModuleGraph; fileIdx: int32): PSym =
     rawMessage(errInvalidModuleName, result.name.s)
 
   result.info = newLineInfo(fileIdx, 1, 1)
-  let pack = getIdent(getPackageName(filename))
+  let
+    pck = getPackageName(filename)
+    pck2 = if pck.len > 0: pck else: "unknown-package"
```

Comment:
Sorry. I'll fix soon.

I think invalid name for package is better than valid one for this.
What do you think?
If invalid name is better, I will register this as another issue.